SearchHistory/test_search_page.py

```diff
@@ -42,3 +43,19 @@ def test_search_by_professional_id(client, make_professional):
     assert professionals[0].profile_id.user_id.first_name == professional.profile_id.user_id.first_name
     assert professionals[0].profile_id.user_id.last_name == professional.profile_id.user_id.last_name
     assert professionals[0].profile_id.city == professional.profile_id.city
+
+
+@pytest.mark.django_db
+def test_redirection_to_professional_page(client, make_professional):
+    professional = make_professional()
+    typeOfjobs_by_pro = TypeOfJob.get_typeofjobs_by_professional(professional_id=professional.professional_id)
+    client.force_login(professional.profile_id.user_id)
+    url = reverse('show professional', args=[PROFESSIONAL_ID])
```

Comment:
Please avoid hardcode
You should test it through the data exist in tests and operations
You should get the url different way